### PR TITLE
Miles of style

### PR DESCRIPTION
**Pull request by jessieay**
* Ran `bundle exec rubocop` and fixed stuff
* Not 100% green but this gets us closer to being able to run rubocop
  locally and get helpful information rather than waiting for Code
  Climate

### CODE REVIEW

File: app/serializers/admin/user_serializer.rb [hunk 1]
```diff
@@ -12,10 +12,10 @@ class Admin::UserSerializer < ActiveModel::Serializer
   )
 
   def created_at
-    object.created_at.iso8601 rescue nil
+    object.created_at.iso8601
```
Comment (harrisj) [inline app/serializers/admin/user_serializer.rb]: I know that rubocop wants us to remove `rescue nil` but in this case I think we need to make sure this doesn't leave us with uncaught exceptions being thrown

Comment (jessieay) [inline app/serializers/admin/user_serializer.rb]: `created_at` and `updated_at` are never nil

File: spec/services/create_c2_proposal_spec.rb [hunk 2]
```diff
@@ -38,28 +42,31 @@
...
+          .and_return(response_double)
 
         expect do
           CreateC2Proposal.new(auction).perform
-          auction.reload
         end
-          .to change { auction.c2_proposal_url }
-            .from('')
-            .to("https://c2-dev.18f.gov/proposals/#{fake_c2_proposal_id}")
+          .to change { auction.reload.c2_proposal_url }
```
Comment (harrisj) [inline spec/services/create_c2_proposal_spec.rb]: Was this test not working before?

File: spec/support/helpers/factory_girl.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
-module TimeHelpers
-  def self.quartile_minute(time)
+module FactoryGirl::Syntax::Methods
```
Comment (harrisj) [inline spec/support/helpers/factory_girl.rb]: Oh, much nicer

File: app/serializers/admin/user_serializer.rb (hunk 1 above)
Comment (harrisj) [inline app/serializers/admin/user_serializer.rb]: Okay then!

**Comment by harrisj** [comment]
This looks good to me, but I was wondering if we are planning to copy over the Rubocop file from the Ruby style guide discussion? Or are we still waiting for that to be finalized?

**Comment by jessieay** [comment]
@harrisj hmmmm I am not sure! we could try? I have my preferred styles (many of which we do not have currently) but am happy to use any style guide as long as we use one!

File: spec/support/helpers/factory_girl.rb (hunk 3 above)
Comment (jessieay) [inline spec/support/helpers/factory_girl.rb]: yes! found this method via https://github.com/thoughtbot/factory_girl/issues/564#issuecomment-157669866

File: spec/services/create_c2_proposal_spec.rb (hunk 2 above)
Comment (jessieay) [inline spec/services/create_c2_proposal_spec.rb]: ok will move back

Comment (jessieay) [inline spec/services/create_c2_proposal_spec.rb]: moved at 2caac27

**Comment by harrisj** [comment]
Okay, let's merge this, but it's possible that if we adopt the central style guide, some of this work will be undone :(

**Comment by jessieay** [comment]
@harrisj if that happens, then I will be on my way to code nirvana aka all my code being deleted YES

**Comment by harrisj** [comment]
You are awaited in Code Valhalla

**Pull request merged**